packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -544,6 +573,12 @@ export const Container = styled.div<ContainerProps>`
         }
       }
 
+      ${ControlsIndicatorWrapper} {
+        transform: translateY(
+          ${({ isSettingsPopoverOpened }) => (isSettingsPopoverOpened ? HOVERED_CONTROLS_INDICATOR_POSITION : 0)}
+        );
+      }
+
```

Comment:
As I said in our meeting - let's consult this with a designer first. Maybe there is a better solution